scripts/deploy-statefulset-test-pods.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Initialization
+SCRIPT_DIR=$(dirname "$0")
+source $SCRIPT_DIR/init-env.sh
+mkdir -p ./temp
+cat ./test-target/local-pod-under-test.yaml | RESOURCE_TYPE="StatefulSet" MULTUS_ANNOTATION=$MULTUS_ANNOTATION $SCRIPT_DIR/mo > ./temp/rendered-local-sateful-pod-under-test-template.yaml
```

Comment:
Be aware there is a typo here (sateful vs stateful). I think it's ok because it is repeated everywhere, but it should be fixed.